WebDriverAgentLib/Categories/XCUIElement+FBWebDriverAttributes.m

```diff
@@ -25,23 +26,19 @@ @implementation XCUIElement (WebDriverAttributesForwarding)
 
 - (XCElementSnapshot *)fb_snapshotForAttributeName:(NSString *)name
 {
-  if (!self.exists) {
```

Comment:
Why remove `self.exists`?

Reply:
because it does extra calls to resolve the element. The below calls anyway would throw stale element exception if the corresponding element does not exist